release/src/main/scripts/build_release_candidate.sh

```diff
@@ -164,7 +165,10 @@ if [[ $confirmation = "y" ]]; then
   git checkout ${RELEASE_BRANCH}
 
   echo '-------------------Generating Python Artifacts-----------------'
+  # Cannot use Gradle because pygradle is hardcoded to tgz (for now)
```

Comment:
Can you link to some context on `(for now)`? It would be useful to determine when this can be removed.

Reply:
OK I see now that actually it should be very easy to add the Gradle rule. To avoid churn on what this script does versus what Gradle does, I'd like to first make this small change (remove the comment), finish the release, then work more on the automation.